### ISSUE TITLE
Resolve illegal characters in eshell on Japanese Windows

### ISSUE THREAD

**Issue by MinoruSekine** [MinoruSekine/dotfiles#131]

**Comment by MinoruSekine** [comment]
`set-buffer-process-coding-system` may help this.

**Comment by MinoruSekine** [comment]
https://w.atwiki.jp/ntemacs/pages/16.html

**Comment by MinoruSekine** [comment]
> https://w.atwiki.jp/ntemacs/pages/16.html

In my little trial, codes below will work.

```lisp
(setq default-process-coding-system '(undecided-dos . utf-8-unix))
```

**Comment by MinoruSekine** [comment]
This has been resolved by #132 .
